app/medInria/resources/authors.html

```diff
@@ -1,20 +1,15 @@
-<b>Authors :</b>
+<b>Authors:</b>
```

Comment:
Here is the list displayed at launch of the app->About->Authors.
Since everyone is defined here, no need to put huge name list in plugins information widgets